taichi/rhi/dx/dx_device.h

```diff
@@ -174,6 +175,15 @@ class Dx11Device : public GraphicsDevice {
 
   DeviceAllocation allocate_memory(const AllocParams &params) override;
   void dealloc_memory(DeviceAllocation handle) override;
+
+  ShaderResourceSet *create_resource_set() final {
```

Comment:
Why raw pointer instead of `unique_ptr`?

Reply:
It is STL, so there is a wrapper function that makes a unique one in the device.h headers (so that we should be able to link to the library from a different compiler, while still having unique pointer access)